src/networkNode.ts

```diff
@@ -179,6 +204,10 @@ export class NetworkNode implements NetworkNodeInterface {
         return;
       };
 
+      // // force connect
+      // if (this.apiInstance) (this.apiInstance as ApiPromise).disconnect();
+      // await this.connectApi();
```

Comment:
Please remove commented code (If its a TODO thing, just add that to a comment )

Reply:
Ahh nice catch, I will remove those lines. Thanks!